core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -610,8 +611,8 @@ class ZkMigrationIntegrationTest {
     new ClusterConfigProperty(key = "listener.security.protocol.map", value = "EXTERNAL:PLAINTEXT,PLAINTEXT:PLAINTEXT"),
   ))
   def testNewAndChangedTopicsInDualWrite(zkCluster: ClusterInstance): Unit = {
-    // Create a topic in ZK mode
-    val topicName = "test"
+    val topic1 = "test1"
+    val topic2 = "test2"
     var admin = zkCluster.createAdminClient()
```

Comment:
I can see there are some adminClient is not closed in this test suite. It might be good if we can close them in this PR or in a separate PR.